templates/adherent_profile/_layout.html.twig

```diff
@@ -0,0 +1,163 @@
+              var contents = findAll(accordion.closest('.accordion-container'), '.accordion');
+              contents.forEach(function(content) {
+                content.classList.remove('opened');
+              });
+
+              if (!visible) {
+                accordion.classList.add('opened');
+              }
+            });
+          });
+
+          findAll(document, '.js--tabs').forEach(function(tab) {
+            tab.addEventListener('click', function(e) {
+              e.preventDefault();
+
+              var target = dom(tab.dataset.target);
+
+              findAll(tab.closest('ul'), '.js--tabs').forEach(function(t) {
+                t.parentNode.classList.remove('active');
+              });
+              tab.parentNode.classList.add('active');
+
+              findAll(target.closest('section'), '.js--tabs-content').forEach(function (content) {
+                content.style.display = 'none';
+              });
+              target.style.display = 'block';
+            });
+          });
+        });
+    </script>
+{% endblock %}
+
+{% block flash_message %}
+    {% for label, messages in app.flashes %}
+        {% for message in messages %}
+            <div class="flash flash--{{ label }}">
+                <div class="flash__inner">{{ message|trans|raw }}</div>
+            </div>
+        {% endfor %}
+    {% endfor %}
+{% endblock %}
+
+{% block content %}
+    <main id="adherent-profile">
+        <div class="l__wrapper">
+            <div class="adherent-profile l__row l__row--v-stretch">
+                <aside class="adherent-profile__aside">
+                    <div class="adherent-profile__status b__nudge--top">
+                        <div class="l__row b__nudge--bottom-large">
+                            <div class="avatar-initials avatar--medium avatar--style-02">
+                                {{app.user.initials}}
+                            </div>
+                            <div class="adherent-profile__id">
+                                <div class="name l__row">
+                                    {{ app.user }}
+                                    {% if app.user.certified %}
+                                        {% include('components/_certification_logo.html.twig') %}
+                                    {% endif %}
+                                </div>
+                                <div class="adhesion-date">Inscrit{{ app.user.female ? "e" }} en {{ app.user.registeredAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM yyyy') }}</div>
+                            </div>
+                        </div>
+                        <ul class="adherent-profile__roles b__nudge--bottom-large">
+                            {% for role in get_adherent_role_labels(app.user) %}
+                                <li>{{ role }}</li>
+                            {% endfor %}
+                        </ul>
+{#                        <div class="adherent-profile__completion b__nudge--bottom-50">#}
```

Comment:
Still need to be commented ?

Reply:
yes, @enzozccr did the integration, but the feature will be done in an other pr, so I leave it here to not loose it